Plate_1_data/README.md

```diff
@@ -0,0 +1,13 @@
+# Plate 1 data 
+
+This module holds all of the analysis for the waves of data from plate 1 of the Interstellar collaboration project.
+
+## The waves for the Interstellar plate 1
+
+Currently, there are three waves of data analyzed coming from [plate 1](Plate_1_data/):
+
+| Module | Wave title | Description |
+| :---- | :----- | :---------- |
+| [0.wave1_data](Plate_1_data/0.wave1_data/) | Nuclei Dilaion Optimization | Using the part of the plate with two channels, Hochest and Gasdermin-D antibody, we perform a CellProfiler pipeline with various nuclei dilation to assess which dilation would allow us to identify the Gasdermin-D around each nuclei when going through pyroptosis. |
```

Comment:
```suggestion
| [0.wave1_data](Plate_1_data/0.wave1_data/) | Nuclei Dilation Optimization | Using the part of the plate with two channels, Hoechst and Gasdermin-D antibody, we perform a CellProfiler pipeline with various nuclei dilation (25, 50, 75 pixels) to test if Gasdermin staining is closer to a simulated plasma membrane in conditions that would induce pyroptosis |
```

Please double check this suggestion

Reply:
That works to make the goal more clear! I have made the change now!